src/plugins/ietf/ietf_interface.c

```diff
@@ -14,13 +15,14 @@
  */
 
 #include <stdio.h>
+#include <iomanip>
 #include <sys/socket.h>
 
-#include <vom/interface.hpp>
 #include <vom/om.hpp>
+#include <vom/interface.hpp>
+#include <vom/interface_cmds.hpp>
```

Comment:
This header only exist for unit testing in VOM. You can not rely on it.

Reply:
ietf_interface.c was the first one I did and I used bind cmds there. We should rewrite it again.